qiita_db/support_files/patches/37.sql

```diff
@@ -0,0 +1,36 @@
+-- Apr 18, 2016
+-- Adding phred_offset to split libraries
+
+DO $do$
+DECLARE
+    cmd_id            bigint;
+BEGIN
+    -- selecting command_id of interest
+    SELECT command_id FROM qiita.software_command WHERE name = 'Split libraries FASTQ' INTO cmd_id;
+
+    -- adding new parameter
+    INSERT INTO qiita.command_parameter (command_id, parameter_name, parameter_type, required, default_value)
+        VALUES (cmd_id, 'phred_offset', 'string', False, '');
+
+    -- updating all current artifacts that were generated with this command
+    UPDATE qiita.artifact
+        SET command_parameters = (
+            substring(command_parameters::text FROM 0 FOR char_length(command_parameters::text)) || ',"phred_offset":""}'
+        )::json
+        WHERE artifact_id=2;
+
+    -- updating the per sample FASTQ defaults
```

Comment:
This comment is misleading because it is updating all the "Split libraries FASTQ defaults" not only the per sample FASTQ